ui/push-health/GroupedTests.jsx

```diff
@@ -34,6 +47,23 @@ class GroupedTests extends PureComponent {
     });
   };
 
+  retriggerAll = (times, group) => {
```

Comment:
This function should be called `retriggerSelected` or `retriggerChecked`.

I would store which items are checked in the `state` rather than `LocalStorage`.  Just use `LocalStorage` to store which items have been marked as `investigated`.